pkg/client/gst/gst.c

```diff
@@ -118,7 +118,7 @@ void gstreamer_receive_push_buffer(GstElement *pipeline, void *buffer, int len,
   GstElement *src = gst_bin_get_by_name(GST_BIN(pipeline), element_name);
 
   if (src != NULL) {
-    gpointer p = g_memdup(buffer, len);
+    gpointer p = g_memdup2(buffer, len);
```

Comment:
IIRC doing this breaks the docker build so I've left it deprecated for now (it's just a warning on mac AFAIK)